polyfill/spatial-navigation-polyfill.js

```diff
@@ -555,8 +555,8 @@
       }
     }
 
-    return (minDistanceElements.length > 1 && distanceFunction === getAbsoluteDistance) ?
-      getClosestElement(currentElm, minDistanceElements, dir, getEuclideanDistance) : minDistanceElements[0];
+    return (minDistanceElements.length > 1 && distanceFunction === getAbsoluteDistance)
+      ? getClosestElement(currentElm, minDistanceElements, dir, getEuclideanDistance) : minDistanceElements[0];
```

Comment:
Could I know the reason you change this line break?
I intended this because of "https://eslint.org/docs/rules/multiline-ternary"